### DISCUSSION TITLE
【灌水】感觉录像店的老板很怪

### DISCUSSION THREAD

**Discussion by convant** [share121/inter-knot#1298] [其他]
![微信图片_20240811121322](https://github.com/user-attachments/assets/3baeffbc-0957-420f-803c-b6e248a3e091)
事情是这样
我逛了很多遍录像店，发现店长的员工仓库基本就没打开过，都知道的，人的好奇心是无穷的，我就准备偷偷瞄一眼里面的东西，就像之前在街上看见的翻垃圾桶的小灰毛一样，但是我的伟大计划被旁边的收营员邦布拦住了，我就在蹲角落里准备在里面的邦布打开窗口的时候用手机放大偷看，然后就看见了房间里录像带没几个，显示屏倒是有一堆，这两位不会在做什么违法交易吧。比竟之前录像店都没怎么进录像，这段时间突然就进的多了起来，各位有什么看法吗

**Comment by Jiucheng258** [comment]
翻垃圾桶的小灰毛？！？你在哪看见她的，我需要具体细节，速速速。

**Comment by su2569** [comment]
翻垃圾桶的小灰毛？
那你有看见白色不明飞行物吗？

**Comment by Xieleng42** [comment]
应该是最近才进货吧，显示屏应该只是为了评鉴录像带（自我认为）

**Comment by ZhuYuan-148** [comment]
店长人还是很好的，之前还邀请我去看演唱会…
